### ISSUE TITLE
[Feature Request] Support for blob and data

### ISSUE THREAD

**Issue by asheroto** [goclone-dev/goclone#17]
Hey there, thanks for this, looks great so far!

Got this error while downloading a site...

```
Img found --> blob:https://twitter.com/c86b1e90-025d-4bfa-9844-adb20bc84bf1
Extracting -->  blob:https://twitter.com/c86b1e90-025d-4bfa-9844-adb20bc84bf1
panic: Get "blob:https://twitter.com/c86b1e90-025d-4bfa-9844-adb20bc84bf1": unsupported protocol scheme "blob"
```

and it died.  Feature request, support for "blob" and "data"

```
unsupported protocol scheme "data"
```

I added in a string replace for now but I'm not good enough at go to be worthy of a PR yet. :-)

Thanks!

**Comment by mesaglio** [comment]
Hi @asheroto, I think this issue is resolved on #23.
If you like, please validate with master version.

**Comment by asheroto** [comment]
Okay, thank you!
